dokka-subprojects/analysis-kotlin-symbols/src/main/kotlin/org/jetbrains/dokka/analysis/kotlin/symbols/translators/DefaultSymbolToDocumentableTranslator.kt

```diff
@@ -128,16 +127,16 @@ internal class DokkaSymbolVisitor(
         moduleFiles: Set<KtFile>
     ): DPackage {
         val dri = getDRIFromPackage(packageSymbol)
-        val scope = packageSymbol.getPackageScope()
-        val callables = scope.callables.toList().filterSymbolsInSourceSet(moduleFiles)
-        val classifiers = scope.classifiers.toList().filterSymbolsInSourceSet(moduleFiles)
+        val scope = packageSymbol.packageScope
+        val callables = scope.callables.filterSymbolsInSourceSet(moduleFiles)
```

Comment:
`callables` and `classifiers` should be converted to `List` here, to avoid multiple recreation of sequences.